discovery/file/file.go

```diff
@@ -23,13 +23,23 @@ func (s *FileDiscoveryService) Initialize(path string, heartbeat int) error {
 	return nil
 }
 
+func generateFromData(data []byte) []string {
```

Comment:
Another small nit: Could this be called something different?

For instance, `parse` could make sense, or `parseContent`, `parseFileContent`, ...

`Data` is very generic and overall I try to use that word in function names (same as `object`, `items`, ...)